vtex/faststore
# feat!: Adds `Newsletter` and its child components

## What's the purpose of this pull request?

This PR intends to add the `Newsletter` and its child components to `@faststore/ui`.

**BREAKING CHANGE**: We are removing the `__experimentalNewsletterAddendum` override since this PR adds the `NewsletterAddendum` component to `@faststore/ui`.

## How to test it?

- Run `yarn workspace site dev` on `faststore` root directory and check the `Newsletter` page or use the [Doc Preview Link](https://faststore-site-git-feat-fs-247-move-newsletter-23852e-faststore.vercel.app/components/organisms/newsletter).

### Starters Deploy Preview

https://github.com/vtex-sites/starter.store/pull/346
